api/graphql/schema.graphql

```diff
@@ -929,10 +929,11 @@ type Query {
       scopes: ["notDraft", "availableForOpportunities"]
     )
     @guard
-    @can(ability: "viewAny")
+    @can(ability: "view", resolved: true)
```

Comment:
Yeah, I'm really not sure. This was the only way I could figure out how to do this since we do need to check each one individually to see if they are part of the team 🤔 

The only other way would be to force the query to only return any `PoolCandidates` are are part of the current team instead of checking on the policy.